src/main/java/com/cybozu/kintone/client/module/parser/Parser.java

```diff
@@ -5,7 +5,7 @@
 import com.google.gson.JsonElement;
 
 public class Parser {
-	private static final Gson gson = new Gson();
```

Comment:
Why `gson` is static ?  it seems that `gson` should not be static

Reply:
I've updated this in commit [d714b76](https://github.com/kintone/kintone-java-sdk/pull/47/commits/d714b76559aec600ac81d7cbeb3599c41674447e). There's a lot of legacy code to refactor. Thanks for your review.